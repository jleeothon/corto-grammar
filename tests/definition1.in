int8 a: 0
