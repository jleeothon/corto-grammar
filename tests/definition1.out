(program (statement (definition (declaration (typeLabel (identifier int8)) a) : (initializerList (initializerElem (initializerValue (literal 0))))) \n))
